app/src/main/java/com/puzzlebench/cleanMarvelKotlin/presentation/mvp/CharacterPresenter.kt

```diff
@@ -1,16 +1,45 @@
 package com.puzzlebench.cleanMarvelKotlin.presentation.mvp
 
+import com.puzzlebench.cleanMarvelKotlin.domain.usecase.GetCharacterDetailServiceUseCase
 import com.puzzlebench.cleanMarvelKotlin.domain.usecase.GetCharacterServiceUseCase
 import com.puzzlebench.cleanMarvelKotlin.presentation.base.Presenter
+import com.puzzlebench.cleanMarvelKotlin.utils.bus.RxBus
+import com.puzzlebench.cleanMarvelKotlin.utils.bus.observer.OnCharacterPressedBusObserver
 import io.reactivex.android.schedulers.AndroidSchedulers
 import io.reactivex.disposables.CompositeDisposable
 import io.reactivex.schedulers.Schedulers
 
-class CharacterPresenter(view: CharacterView, private val getCharacterServiceUseCase: GetCharacterServiceUseCase, private val subscriptions: CompositeDisposable) : Presenter<CharacterView>(view) {
+class CharacterPresenter(view: CharacterView,
+                         private val getCharacterServiceUseCase: GetCharacterServiceUseCase,
+                         private val getCharacterDetailServiceUseCase: GetCharacterDetailServiceUseCase,
+                         private val subscriptions: CompositeDisposable) : Presenter<CharacterView>(view) {
 
     fun init() {
         view.init()
         requestGetCharacters()
+        RxBus.subscribe(view.activity, object : OnCharacterPressedBusObserver() {
+            override fun onEvent(value: OnCharacterPressedBusObserver.OnCharacterPressed) {
+                val subscription = getCharacterDetailServiceUseCase.invoke(value.id).
+                        subscribeOn(Schedulers.io()).observeOn(AndroidSchedulers.mainThread())
+                        .subscribe({ character ->
+                    var detailsLink = ""
```

Comment:
use constants please

Reply:
I would put the dot on the new line 
```suggestion
                    getCharacterDetailServiceUseCase.invoke(value.id)
                    .subscribeOn(Schedulers.io())
                    .observeOn(AndroidSchedulers.mainThread())
```

Reply:
done!